Purple Dot Pre-Order button/scripts.js

```diff
@@ -1,4 +1,6 @@
 let selectedVariant = Tapcart.variables.product.selectedVariant.id;
+let currency = Tapcart.variables.cart.currency;
+const supportedCurrencies = [];
```

Comment:
This name makes it sound like it won't allow anything if the list is empty. Can we think of a better name? Maybe something like `onlyAllowCurrencies`?

Reply:
Good suggestion, updated.